feg/gateway/services/s8_proxy/servicers/mock_pgw/cs.go

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
maybe i will move this mock_pgw *folder* somewhere else (maybe to gtp folder) or use a different name to indicate it is for test